godot-macros/src/lib.rs

```diff
@@ -318,6 +318,34 @@ use crate::util::ident;
 /// for more information and further customization.
 ///
 /// This is very similar to [GDScript's `@tool` feature](https://docs.godotengine.org/en/stable/tutorials/plugins/running_code_in_the_editor.html).
+///
+///
+/// # Class Name Aliasing
+///
+/// You may want to have structs with the same name.  With rust, this is
+/// allowed using `mod`.  However in GDScript, there are no modules,
+/// namespaces, or any such diambiguation.  Therefore, you need to
+/// change the names before they can get to Godot.  You can use the
+/// `rename` key while defining your `GodotClass` for this.
```

Comment:
Use a bit wider lines if possible (~120 chars).

Also, some typos:
* rust -> Rust
* diambiguation
* no need for double space after full stop